clippy_lints/src/casts/unnecessary_cast.rs

```diff
@@ -52,7 +52,7 @@ pub(super) fn check<'tcx>(
             UNNECESSARY_CAST,
             expr.span,
             format!(
-                "casting raw pointers to the same type and constness is unnecessary (`{cast_from}` -> `{cast_to}`)"
+                "casting raw pointers to the same type and constness is unnecessary (`{cast_from}` -> `{cast_to}` :D)"
```

Comment:
:eyes: Don't forget to remove this again :)

Reply:
Or we could create an RFC suggesting that all lint messages should from now on include ASCII emojis :P

(Yep, I'll undo these changes, the commit also has a big "do not merge" in the name)